packages/core/js-client/createVersionFile.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
I would at least add // @ts-check in this js file if it's simpler to just use js in this script

Reply:
Added @ts-check. But still i do not explicitly typecheck this file cus it requires too many changes in build output which i consider not feasible for the sake of 1 file.